go/vt/vttablet/tabletserver/planbuilder/plan.go

```diff
@@ -185,7 +185,13 @@ func Build(statement sqlparser.Statement, tables map[string]*schema.Table, isRes
 		// DDLs and some other statements below don't get fully parsed.
 		// We have to use the original query at the time of execution.
 		// We are in the process of changing this
-		plan = &Plan{PlanID: PlanDDL}
+		var fullQuery *sqlparser.ParsedQuery
+		fullQuery = nil
```

Comment:
nit: this is not really needed, since it is the default value references get

Reply:
removed